Strato/Licences.rb

```diff
@@ -0,0 +1,17 @@
+require 'fileutils'
+
+class Licences
+
+	def self.generateCleanAcknowledgementsPlist(target, path)
```

Comment:
It's fine IMO 👍 
Can we just have a short documentation for the function explaining what each parameter is (at least their types) ?